2.4-micro/Dockerfile.fedora

```diff
@@ -3,7 +3,7 @@ FROM quay.io/fedora/fedora:rawhide AS build
 RUN mkdir -p /mnt/rootfs
 RUN MICRO_PKGS="coreutils-single glibc-minimal-langpack" && \
     INSTALL_PKGS="$MICRO_PKGS httpd-core mod_ssl findutils hostname nss_wrapper-libs fedora-logos-httpd" && \
-    dnf install --installroot /mnt/rootfs $INSTALL_PKGS --releasever 36 --setopt install_weak_deps=false --nodocs -y && \
+    dnf install --installroot /mnt/rootfs --use-host-config $INSTALL_PKGS --releasever 39 --setopt install_weak_deps=false --nodocs -y && \
```

Comment:
Only one question, why `releasever 39`?

Reply:
Because we are currently tracking rawhide

Reply:
Is it a goal for all of Fedora micro images to track rawhide?

Reply:
Not a goal but that is what we started with as the packages were not available anywhere else.
We can decide on pinning a specific Fedora version in a separate PR if that is something we want to do.